pose_format/utils/holistic.py

```diff
@@ -85,23 +87,26 @@ def holistic_hand_component(name, pf="XYZC"):
     return PoseHeaderComponent(name=name, points=HAND_POINTS, limbs=HAND_LIMBS, colors=hand_colors, point_format=pf)
 
 
-def holistic_components(pf="XYZC"):
+def holistic_components(pf="XYZC", face_num=468):
     return [
         PoseHeaderComponent(name="POSE_LANDMARKS", points=BODY_POINTS, limbs=BODY_LIMBS,
                             colors=[(255, 0, 0)], point_format=pf),
-        PoseHeaderComponent(name="FACE_LANDMARKS", points=FACE_POINTS, limbs=FACE_LIMBS,
+        PoseHeaderComponent(name="FACE_LANDMARKS", points=[str(i) for i in range(face_num)], limbs=FACE_LIMBS,
```

Comment:
if using a lambda, you can pass `additional_face_points` and use `FACE_POINTS(additional_face_points)`